db/seeds.rb

```diff
@@ -46,20 +46,24 @@ def generate_items
 
   def generate_users
     @users = User.create([
-            { full_name: "Rachel Warbelow", email: "demo+rachel@jumpstartlab.com", password: "password"},
-            { full_name: "Jeff Casimir", email: "demo+jeff@jumpstartlab.com", password: "password", display_name: "j3"},
-            { full_name: "Jorge Tellez", email: "demo+jorge@jumpstartlab.com", password: "password", display_name: "novohispano"},
-            { full_name: "Josh Cheek", email: "demo+josh@jumpstartlab.com", password: "password", display_name: "josh", role: 1}
+            { id: 1, full_name: "Rachel Warbelow", email: "demo+rachel@jumpstartlab.com", password: "password"},
```

Comment:
Line is too long. [112/80]<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Space inside } missing.